pkg/capabilities/consensus/ocr3/factory.go

```diff
@@ -59,6 +62,11 @@ func (o *factory) Start(ctx context.Context) error {
 
 func (o *factory) Close() error {
 	return o.StopOnce("OCR3ReportingPlugin", func() error {
+		err := (*o.capabilityRegistry).Remove(o.capability)
```

Comment:
The factory object is not adding it to the Registry, is it? I think we should handle removal in the same object that Add()-ed it in the first place (where is that?)

Reply:
No, it's [here](https://github.com/smartcontractkit/chainlink-common/blob/CAPPL-205/pkg/capabilities/consensus/ocr3/ocr3.go#L99). I thought about that too, agreed that it makes sense to move.